app/src/main/java/org/oppia/android/app/help/HelpActivity.kt

```diff
@@ -47,11 +51,17 @@ class HelpActivity :
     )
     selectedFragment =
       savedInstanceState?.getString(SELECTED_FRAGMENT_SAVED_KEY) ?: FAQ_LIST_FRAGMENT_TAG
-    val extraHelpOptionsTitle = savedInstanceState?.getString(HELP_OPTIONS_TITLE_SAVED_KEY)
+    selectedDependencyIndex =
+      savedInstanceState?.getInt(THIRD_PARTY_DEPENDENCY_INDEX_SAVED_KEY) ?: 0
```

Comment:
This can be shifted to single line.

Reply:
Actually, it is crossing the max 100 char limit when I am trying to fit it into a single line

Reply:
Ack